src/dmd/traits.d

```diff
@@ -824,7 +824,7 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         e.ident == Id.getVirtualMethods ||
         e.ident == Id.getVirtualFunctions)
     {
-        if (dim != 2)
+        if (dim != 2 && !(dim == 3 && e.ident == Id.getOverloads))
             return dimError(2);
```

Comment:
codecov shows this line isn't covered in the test suit.

Reply:
Question: Should this be covered by a separate test file in the fail_compilation folder, or is a `static assert(!__traits(compiles))` in Test16206.d good enough?

Reply:
Scratch that. Looking at the existing tests, I was able to notice a pattern. :p